apps/web/src/lib/i18n.ts

```diff
@@ -0,0 +1,45 @@
+import { i18n } from '@lingui/core';
+import { IS_PREVIEW, IS_PRODUCTION } from 'data/constants';
+import dayjs from 'dayjs';
+import { en } from 'make-plural/plurals';
+
+export const supportedLocales: Record<string, string> = {
+  en: 'English'
+};
+
+if (!IS_PRODUCTION || IS_PREVIEW) {
+  supportedLocales.qaa = 'PseudoLanguage';
+}
+
+const defaultLocale = 'en';
+
+i18n.loadLocaleData({
+  en: { plurals: en },
+  qaa: { plurals: en }
+});
+
+import('dayjs/locale/cs');
```

Comment:
we can move to top

Reply:
Removed it for now because it was left there by mistake. Those will be needed again when we start adding languages.